src/cls/IPM/Repo/Oras/Definition.cls

```diff
@@ -0,0 +1,183 @@
+Class %IPM.Repo.Oras.Definition Extends %IPM.Repo.Definition
+{
+
+Parameter DISPLAYNAME As STRING = "ORAS";
+
+Parameter MONIKER As STRING = "oras";
+
+Property URL As %String(MAXLEN = 2048) [ Required ];
```

Comment:
Rather than repeating a bunch of properties, can this extend the remote repo definition or move the common properties to a common super class in the %IPM.General package?

Reply:
Yes, I'll go refactor by creating IPM.Repo.HTTP parent classes that both IPM.Repo.Remote and IPM.Repo.ORAS can inherit from.

Reply:
Done.